File: java/moduleA/src/main/java/dev/potaro/multiprojectexample/a/MultiProjectExampleA.java
package dev.potaro.multiprojectexample.a;

import org.bukkit.plugin.java.JavaPlugin;

public final class MultiProjectExampleA extends JavaPlugin {

    @Override
    public void onEnable() {
        getLogger().info("Enabled test plugin A");
    }

    @Override
    public void onDisable() {
        getLogger().info("Disabled test plugin A");
    }
}

File: kotlin/moduleA/src/main/kotlin/dev/potaro/multiprojectexample/a/MultiProjectExampleA.kt
package dev.potaro.multiprojectexample.a

import org.bukkit.plugin.java.JavaPlugin

class MultiProjectExampleA : JavaPlugin() {

    override fun onEnable() {
        logger.info("Enabled test plugin A")
    }

    override fun onDisable() {
        logger.info("Disabled test plugin A")
    }
}
